experiment_scheduler/master/master.py

```diff
@@ -343,7 +369,7 @@ def serve():
     If an anomaly action erupt, kill process monitor before close master object
     :return: None
     """
-
+    initialize_db()
```

Comment:
Can this initialized at __init__ in Master class?

Reply:
I'm not sure run DB in __init__. how about letting it be?

Reply:
This initialize_db is for sqlite file creation. I misunderstood